examples/validating-public-input/sp1/fibonacci/program/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+version = "0.1.0"
+name = "fibonacci-program"
+edition = "2021"
+
+[dependencies]
+sp1-zkvm = { git = "https://github.com/succinctlabs/sp1.git", rev = "v1.0.8-testnet"  }
```

Comment:
We should have this match the supported SP1 version in the docs `rev = "v1.0.8-testnet"` -> `tag = "v1.0.1"`

Reply:
Makes sense, updated in [1c9016d](https://github.com/yetanotherco/aligned_layer/pull/1245/commits/1c9016df9f678a93d122ded62a688f13810e5d73)